pylint/checkers/variables.py

```diff
@@ -1400,7 +1406,17 @@ def _check_consumer(
         if found_nodes is None:
             return (VariableVisitConsumerAction.CONTINUE, None)
         if not found_nodes:
-            self.add_message("used-before-assignment", args=node.name, node=node)
+            if node.name in current_consumer.consumed_uncertain:
+                confidence = CONTROL_FLOW
+            else:
+                # Set to HIGH in future?
+                confidence = None
```

Comment:
``HIGH`` only means we do not use inference, I think we can use ``HIGH``.
```suggestion
                confidence = HIGH
```